### DISCUSSION TITLE
onze-toetsen-zijn-betrouwbaar

### DISCUSSION THREAD

**Discussion by sannevanseeventer** [fdnd/docs.fdnd.nl#80] [Reflectie]
Wij twijfelen hier soms aan, omdat je niet specifiek wordt getoetst op de code. Soms hebben wij het idee dat de softs kills zwaarder wegen dan de hard skills. We hebben wel vertrouwen in het oordeel van de docenten.

**Comment by ju5tu5** [comment]
Opgenomen in de lopende tekst.
